sparse/file.go

```diff
@@ -220,10 +220,5 @@ func GetFiemapRegionExts(file FileIoProcessor, interval Interval) ([]Extent, err
 		lastExtStart = ext.Logical
 	}
 
-	// last ext should have the FIEMAP_EXTENT_LAST set, otherwise we don't get all exts
-	if exts[len(exts)-1].Flags&FIEMAP_EXTENT_LAST == 0 {
```

Comment:
The flag is for the file's last ext, not region's last ext. Code used to get entire file's exts and we had suspicious corruption issue during stress test, it was added to ensure it was not due to incomplete exts.